examples/src/main/java/com/oddin/oddsfeed/examples/Main.java

```diff
@@ -34,6 +34,7 @@ public static void main(String[] args) {
         // You need to properly set your access token in order to access API and Feed features.
         // Please check with support that your IP was whitelisted.
         OddsFeedConfiguration configuration = OddsFeed.getOddsFeedConfigurationBuilder()
```

Comment:
Plese also update property 'oddsFeedVersion' in example/gradle.build file with new git tag.